tests/syntax-tests/test_custom_assets.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+set -o errexit -o nounset -o pipefail
+
+### HELPER VARS
+
+custom_syntaxes_dir="$(bat --config-dir)/syntaxes"
+
+installed_custom_syntax="${custom_syntaxes_dir}/BatTestCustomAssets.sublime-syntax"
```

Comment:
This shell script will modify the local environment. Even if it cleans up in the end, there might be changes to the environment (e.g. existing serialized custom assets will be lost).

I wonder if we should add a comment at the top or rename it slightly (`ci_pipeline_test_…`) to make sure that this is not intended to be ran locally.

Another way would be to perform everything in a temporary directory. By setting `BAT_CONFIG_DIR`/`BAT_CACHE_DIR` (or respective command line options) appropriately.

A third option could be to move everything inside a `script:` block in the CI configuration (maybe inside a new yaml file).

Reply:
Making use of `BAT_CONFIG_DIR`/`BAT_CACHE_DIR` is a great idea. Will fix.

Reply:
I think it is important to be able to run the script locally. That way it is easy to use it for local verification, but it is also easy to work on the script. So the idea to use the env vars was a great idea. It turned me into such a good mode that I also added another test that makes sure `bat cache --clean` removes all files.